aopy/data/base.py

```diff
@@ -142,6 +142,32 @@ def load_preproc_exp_data(preproc_dir, subject, te_id, date, verbose=True, cache
         warnings.warn(f"Preprocessing errors found in {filename}:\n{metadata['preproc_errors']}")
     return data, metadata
 
+def load_preproc_emg_data(preproc_dir, subject, te_id, date, verbose=True, cached=True):
```

Comment:
you can remove the verbose parameter and the error printing unless you think there's a need for it